builder/module-docs.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, lib, ... }:
+let
+  # Inspired by https://github.com/astro/microvm.nix/blob/e8d5f12b834a59187c7ec147a8952a0567f97939/pkgs/doc.nix
+  extraConfig = { lib, ... }: {
+    options._module.args = lib.mkOption {
+      internal = true; # Hide `_module` from the docs
+    };
+
+    config._module = {
+      check = false; # Don't check for unset variables
+      args = {
+        inherit pkgs lib; # Provide pkgs and lib to modules
```

Comment:
From my experience with nüschtos search you generally don't need to provide lib.